pkg/metrics/eventcachemetrics/eventcachemetrics.go

```diff
@@ -39,6 +41,12 @@ var (
 		Help:        "The total of errors encountered while fetching process exec information from the cache.",
 		ConstLabels: nil,
 	}, []string{"error"})
+	ProcessCacheTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
It would make sense to name the metric `event_cache_size` I think, and make it a gauge instead of counter. Using a counter here might be not incorrect if the cache size is only increasing until it's reset, but that's an implementation detail of the cache.